backend/src/ee/services/license/licence-fns.ts

```diff
@@ -40,7 +40,17 @@ export const getDefaultOnPremFeatures = (): TFeatureSet => ({
   secretRotation: true,
   caCrl: false,
   instanceUserManagement: false,
-  externalKms: false
+  externalKms: false,
+  rateLimits: {
+    readLimit: 60,
+    publicEndpointLimit: 30,
+    writeLimit: 200,
```

Comment:
what is the difference between write vs creation end points?

Reply:
hmmm based on existing usage, creationLimit is for org, project, and machine identity creation while writeLimit is for everything else